server/controllers/chartsController.js

```diff
@@ -102,6 +102,11 @@ export const graphDataController = async (dataDb, userAccess, chart_id) => {
           targetTotal: newTargetValue,
         }
       }
+      if (!!newTargetValue && !!studyTotals[constants.TOTALS_STUDY].targetTotal)
+        studyTotals[constants.TOTALS_STUDY].targetTotal =
+          studyTotals[constants.TOTALS_STUDY].targetTotal + newTargetValue
+      if (!!newTargetValue && !studyTotals[constants.TOTALS_STUDY].targetTotal)
+        studyTotals[constants.TOTALS_STUDY].targetTotal = newTargetValue
```

Comment:
you can remove the `!studyTotals[constants.TOTALS_STUDY].targetTotal` from the second conditional by using `if/else`.

```suggestion
      if (!!newTargetValue && !!studyTotals[constants.TOTALS_STUDY].targetTotal) {
        studyTotals[constants.TOTALS_STUDY].targetTotal =
          studyTotals[constants.TOTALS_STUDY].targetTotal + newTargetValue
      } else if (!!newTargetValue) {
        studyTotals[constants.TOTALS_STUDY].targetTotal = newTargetValue
      }
```

Reply:
I think lines 106 & 107 could be consolidated too 🎱 

```diff
      if (!!newTargetValue && !!studyTotals[constants.TOTALS_STUDY].targetTotal) {
-       studyTotals[constants.TOTALS_STUDY].targetTotal =
-         studyTotals[constants.TOTALS_STUDY].targetTotal + newTargetValue
+       studyTotals[constants.TOTALS_STUDY].targetTotal += newTargetValue
      } else if (!!newTargetValue) {
        studyTotals[constants.TOTALS_STUDY].targetTotal = newTargetValue
      }
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/147/commits/3987aca50d0a9aa1847d57af47a391bc245d7a8c